content/home.pml

```diff
@@ -196,4 +196,18 @@
     </div>
   </div>
 
-</div>
+
+  <div class="homepage_item">
+    <a class="divLink" href="/Info/"></a>
+    <div class="hp_pic">
+      <img src="/images/more_info.png" style="width:80px;height:95px;" alt="More info" />
+    </div>
+    <div class="hp_title">
+      Want to know more?
+    </div>
+    <div class="hp_text">
+      Research, roadmap, FAQ
+    </div>
+  </div>
+
+</div>
```

Comment:
Why do we have this twice now? It does appear twice on the landing page in the preview. Looks like an error that should be fixed

Reply:
Yup - it's an error. It may have crept in during the many rebases I did on Monday. Thanks for spotting it. :-)